### ISSUE TITLE
Issue_target_value - random values seen for a new user in the daily goal and applied sections.

### ISSUE THREAD

**Issue by prav29** [se2024-jpg/WolfTrack6.0#9]
A random value is set as daily goal by default. The applied value is also set to 1. Both should be 0 for a new user.

**Comment by joeljogy** [comment]
Agreed about keeping the initial values to 0. While it maybe good to have a default initial daily goal for users (e.g. 5 or 10), it is okay to keep it as 0 and let the users to decide themselves what should be their daily goal.
